alphabase/psm_reader/maxquant_reader.py

```diff
@@ -18,8 +18,8 @@
 warnings.filterwarnings("always")
 
 mod_to_unimod_dict = {}
-for mod_name, unimod_id in MOD_DF[["mod_name", "unimod_id"]].values:
-    unimod_id = int(unimod_id)
+for mod_name, unimod_id_ in MOD_DF[["mod_name", "unimod_id"]].to_numpy():
```

Comment:
is this suffix thing a known pattern? I only now it as private prefix

Reply:
I use it to avoid name clashes, e.g. with built-ins or other variables